examples/xml-to-json-conversion/xml_to_json_conversion.out

```diff
@@ -1,8 +1,6 @@
 #To run the 'xml_to_json_conversion.bal' file, navigate to the directory that contains this file and issue the
 #'ballerina run' command.
 $ ballerina run xml_to_json_conversion.bal
-{"h:Store":{"@xmlns:h":"http://www.test.com","@id":"AST","h:name":"Anne",
-"h:address":{"h:street":"Main","h:city":"94"},"h:code":{"h:item":["4","8"]}}}
-{"Store":{"#id":"AST","name":"Anne","address":{"street":"Main","city":"94"},
-"code":{"item":["4","8"]}}}
+{"h:Store":{"@xmlns:h":"http://www.test.com","@id":"AST","h:name":{"@xmlns:h":"http://www.test.com","#text":"Anne"},"h:address":{"@xmlns:h":"http://www.test.com","h:street":{"@xmlns:h":"http://www.test.com","#text":"Main"},"h:city":{"@xmlns:h":"http://www.test.com","#text":"94"}},"h:code":{"@xmlns:h":"http://www.test.com","h:item":[{"@xmlns:h":"http://www.test.com","#text":"4"},{"@xmlns:h":"http://www.test.com","#text":"8"}]}}}
```

Comment:
This may not render correctly in webpage, that is why this was splited into multiple lines.